bastion/gui/dashboard/main_window.py

```
@@ -0,0 +1,813 @@
+Hidden=false
+X-GNOME-Autostart-enabled=true
+"""
+                with open(desktop_file, "w") as f:
+                    f.write(content)
+
+                # Make executable just in case
+                desktop_file.chmod(0o755)
+            else:
+                if desktop_file.exists():
+                    desktop_file.unlink()
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)